docs/file_formats.md

```
@@ -164,27 +164,28 @@ The PSM section in mzTab files starts with a header line, indicated by the `PSH`
-- `sequence`: The predicted peptide sequence.
+- `sequence`: The predicted unmodified peptide sequence.
 - `PSM_ID`: A monotonically increasing index, serving as a unique identifier for each PSM.
 - `search_engine_score[1]`: The score of this PSM.
 - `spectra_ref`: Unique identifier linking the prediction back to the original spectrum in the input file(s).
-- `opt_ms_run[1]_aa_scores`: Casanovo predicts peptides in an autoregressive fashion, one amino acid at a time. This column contains comma-separated scores of the individual amino acid predictions.
+- `opt_global_aa_scores`: Casanovo predicts peptides in an autoregressive fashion, one amino acid at a time. This column contains comma-separated scores of the individual amino acid predictions.
+- `opt_ms_global_cv_MS:1003169_proforma_peptidoform_sequence`: The predicted modified peptide sequence in ProForma format.
 
 When running Casanovo in _database searching mode_ rather than *de novo* peptide sequencing mode, the PSM section will look slightly differently:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)